rpadmin/admin_test.go

```diff
@@ -222,3 +226,93 @@ func callsForNodeID(calls []testCall, nodeID int) []testCall {
 	}
 	return filtered
 }
+
+func TestAdminAddressesFromK8SDNS(t *testing.T) {
+	adminAPIURL := "http://redpanda-api.cluster.local:19644"
```

Comment:
Can we add another test with https please?

Reply:
fixed